### ISSUE TITLE
Number of Epochs for Fine-tuning Tasks

### ISSUE THREAD

**Issue by srikar2097** [microsoft/i-Code#16]
Hi Authors,
Appreciate your interesting paper UDOP. Wanted to know the specifics on fine-tuning. Appendix C.6 does provide some details but the crucial number of epochs is missing. Specifically:
1. Number of epochs fine-tuned for each downstream task?
2. Are all downstream tasks fine-tuned for the same number of epochs?

**Comment by zinengtang** [comment]
1. Usually, we take checkpoint by the best validation accuracy for testing. But usually, training needs 4 epochs.
2. Not all downstream tasks are fine-tuned for the same epochs. But this would only be notable for TabFact which took us 10 epochs.
